src/azure-cli/azure/cli/command_modules/acs/azuremonitormetrics/addonput.py

```diff
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import json
+from azure.cli.command_modules.acs.azuremonitormetrics.constants import AKS_CLUSTER_API
+from azure.cli.core.azclierror import (
+    UnknownError,
+    CLIError
+)
+
+
+# pylint: disable=line-too-long
+def addon_put(cmd, cluster_subscription, cluster_resource_group_name, cluster_name):
+    from azure.cli.core.util import send_raw_request
+    armendpoint = cmd.cli_ctx.cloud.endpoints.resource_manager
+    feature_check_url = f"{armendpoint}/subscriptions/{cluster_subscription}/resourceGroups/{cluster_resource_group_name}/providers/Microsoft.ContainerService/managedClusters/{cluster_name}?api-version={AKS_CLUSTER_API}"
+    try:
+        headers = ['User-Agent=azuremonitormetrics.addon_get']
+        r = send_raw_request(cmd.cli_ctx, "GET", feature_check_url,
+                             body={}, headers=headers)
```

Comment:
Is this REST request not currently supported in the Python SDK?

Reply:
We have done a bug bash using this version of the CLI within the team and everything looks good with it and hence I would like to release this version of the azure monitor metrics addon in the build CLI otherwise we will have to do a bug bash for the whole team again as the REST api code is critical to the addon enablement.

I think your feedback is valid so I have created a follow up task [here](https://msazure.visualstudio.com/InfrastructureInsights/_workitems/edit/21536128). Can I create a follow up PR for this and merge it in after the May 17th code complete cut-off for build so that this version which has been tested by everyone in our team can go out?

Reply:
OK, sounds good. Perhaps another good option is to try using Code Gen V2 to generate these new commands, which can improve your development efficiency and ensure code standardization and quality. If you are interested in it, you can contact @kairu-ms